aten/src/ATen/core/dispatch/OperatorEntry.cpp

```diff
@@ -157,6 +157,12 @@ void OperatorEntry::registerSchema(FunctionSchema&& schema, std::string&& debug,
   #endif
 }
 
+void OperatorEntry::unsafeSetTags(const std::vector<at::Tag>& tags) {
+  #ifndef C10_MOBILE
+    tags_ = std::move(tags);
+  #endif
```

Comment:
I'm not that good with C++, why is this a std::move?

Reply:
it avoids copy